### ISSUE TITLE
explore if add-on discovery with more infomation increases users likeliness to install

### ISSUE THREAD

**Issue by designakt** [mozilla/addons#143]
Currently we describe add-ons with a short, active title and a users quote.
I wonder if a short copy about the add-ons functionality might increase users likeliness to install add-ons. (Not necessarily because everyone will read the text, but also because users could feel safer because they know they can get more information easily.)
Or maybe we could indicate somewhere on the page that all add-ons on here are very high quality. e.g. star on every add-on, or as a header.

**Comment by designakt** [comment]
@devaneymoz let's see what we can come up with.

**Comment by devaneymoz** [comment]
How specifically should we approach this? If we come up with some additional copy to contextualize and pitch the add-ons, are we just planning to mock it up for internal review and then decide if it should be implemented or not? Or is the idea to put it in front of users for A/B testing?

**Comment by designakt** [comment]
First would be to mock them up to see for ourselves. But doing A/B testing on it would definitely be better. Maybe we can also get Michelle's opinion on it.

Current state:
![screen shot 2016-06-23 at 15 48 10](https://cloud.githubusercontent.com/assets/8145891/16305770/861c4866-395b-11e6-8460-c563080ae035.png)

Here is my first take on descriptive copy:
![screen shot 2016-06-23 at 15 59 18](https://cloud.githubusercontent.com/assets/8145891/16305772/8752c4c6-395b-11e6-8a52-f1f5fb570c94.png)
(Might need other quotes as social proof that focus more on saying it is awesome than on what it does.)
I also shortened the name of awesome screenshot as it is very odd that they have description in their "name". (Seams like a hack to me.)

**Comment by devaneymoz** [comment]
Here's a stab at some extra copy for each add-on (no copy for the themes for now, but I'm open to experimenting with that too). 

**Block ads** with AdBlock Plus
One of the most popular add-ons of all time, Ad Block Plus puts you in full control of your online advertising experience. Remove ads, malware domains, and more. 

_“This add-on is amazing. Blocks all annoying ads. 5 stars.”_ - rany

**Take screenshots** with Awesome Screenshot Plus
More than just a basic screenshot tool, Awesome Screenshot Plus lets you annotate images with custom copy and graphics, plus the ability to store and share your visuals. 

_“The best. Very easy to use.”_ - meetdak

**Up your emoji game** with Emoji Cheatsheet
A simple search function helps you to find an emoji for any occasion. Works beautifully with Twitter, Facebook, GitHub, and other social sites.  

_“Very simple interface and easy to use. Thank you.”_ - shadypurple

**Download videos** with Video DownloadHelper
Easily grab videos from YouTube, Dailymotion, Facebook, Periscope, Vimeo, Twitch, and most other video sites with this incredibly simple download tool.

_“Download videos in a single click.”_ - Carpe Diem

**Comment by brampitoyo** [comment]
I do have a hunch that describing what an add-on does in plain language will increase conversion, but it may also result in the area becoming too wordy.

What might be a good idea is do an A/B test where we compare:

* Quote only, no description (control scenario, since this is what we are initially proposing?)
* Description only, no quote
* No description, no quote
* Both description and quote (what we are proposing in this issue)

**Comment by andymckay** [comment]
I love the idea of A/B testing. I would like to explore how we'd set this up from an Engineering point of view and tracking metrics around that testing. We haven't done that on AMO and we totally should be.

**Comment by mheubusch** [comment]
I'm not sure this is feasible, but I think bringing forward the star ratings with number of people who starred would help communicate a lot about the value of the apps.  I think shopping sites are a useful paradigm, because we are asking people to select additional products.  For shopping, I know that many people rely on the combination of rating + number of people who rated as independent verification of relevance and quality. Here's one study that shows people are more likely to click on listings with stars vs those without.  Because many of the people who see this panel in the browser do not yet have extensions yet, introducing them with feedback from other users reinforces that 1. many people do use them (popularity) and 2. they see value in them (quality).    I have some feedback on the structure/approach to copy and attached a picture to show.  
![screen shot 2016-06-28 at 2 50 59 pm](https://cloud.githubusercontent.com/assets/20115914/16430191/09df345a-3d40-11e6-8dcb-cf071c14ad04.png)

**Comment by mheubusch** [comment]
Sorry - forgot link to ratings article: http://baymard.com/blog/user-perception-of-product-ratings

**Comment by mheubusch** [comment]
One more thought I forgot to add - we are asking users to commit to installing the add-on in this page, so I think we need to provide enough information to make a decision.  I realize this means more info on the page, but if we are measuring success by number of people who install an add-on from the page, I think we need to at least test having more info than is already there.

**Comment by devaneymoz** [comment]
Thanks, @mheubusch. But what claims in copy can't be supported?

**Comment by mheubusch** [comment]
@devaneymoz  i meant this: "puts you in full control of your online advertising experience"
sorry I wasn't clear - i meant that we as Firefox don't want to claim that someone else's product could give you full control - we want to help people protect themselves online but have no way to guarantee (or even really measure) what full control means.

**Comment by devaneymoz** [comment]
Ah, I see @mheubusch , that makes sense. Would removing "full" solve the issue form your pov? I feel it's an accurate characterization to say it puts users "in control" of their advertising experience.

**Comment by designakt** [comment]
Looks like that is already in product through the descriptions we added to disco pane.
